packages/buttons/stories/4-ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Story, Meta } from '@storybook/react';
+import { Button, ButtonGroup } from '@zendeskgarden/react-buttons';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Buttons/ButtonGroup',
+  component: ButtonGroup
```

Comment:
Should we provide the `Button` component as a `subComponent`?

Reply:
I'm leaning towards no because `Button` isn't a subcomponent that uses dot notation? I did not see  additional documentation by doing:

```jsx
export default {
  title: 'Components/Buttons/ButtonGroup',
  component: ButtonGroup,
  subcomponents: {
    Button
  }
} as Meta;
```